mmdet/models/detectors/glip.py

```diff
@@ -58,8 +58,8 @@ def remove_punctuation(text: str) -> str:
         str: The text with punctuation removed.
     """
     punctuation = [
-        '|', ':', ';', '@', '(', ')', '[', ']', '{', '}', '^', '\'', '\"', '’',
```

Comment:
This part should not need to be modified, right?

Reply:
Reverted this diff in https://github.com/open-mmlab/mmdetection/pull/10648/commits/e95c7063bf1a7fce6f21ba748cd3415ece5167aa 🙆🏻‍♂️